src/test/java/hudson/scm/SubversionSCMTest.java

```diff
@@ -1706,4 +1709,34 @@ private void invokeTestPollingExternalsForFile() throws Exception {
         // should detect change
         assertTrue(p.poll(StreamTaskListener.fromStdout()).hasChanges());
     }
+    
+    public void testUseCommitTimes() throws Throwable {
```

Comment:
When you create tests for a particular JIRA issue, it worth to reference it via comments or <code>@Issue</code> annotation